client/components/inventory.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+var Inventory = (props) => {
+
+  return (
+    <div class="size" title="{props.count}">
+      {props.size}
```

Comment:
and rather than {props.size} you can write {size}

Overall this component is straight forward and solid. Just wanted to share something I learned and began applying to reduce the characters I type.